Tiltfile

```diff
@@ -69,6 +70,21 @@ def deploy_capi():
             if kb_extra_args:
                 patch_args_with_extra_args("capi-kubeadm-bootstrap-system", "capi-kubeadm-bootstrap-controller-manager", kb_extra_args)
 
+# deploy CAAPH
+def deploy_caaph():
+    version = settings.get("caaph_version")
+
+    caaph_uri = "https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/releases/download/{}/addon-components.yaml".format(version)
+    cmd = "curl --retry 3 -sSL {} | {} | {} apply -f -".format(caaph_uri, envsubst_cmd, kubectl_cmd)
+    local(cmd, quiet = True)
+    if settings.get("extra_args"):
+        extra_args = settings.get("extra_args")
+        if extra_args.get("helm"):
```

Comment:
We're testing again on line 84 to see if `extra_args.get("helm")` returns `None`, so I don't think this line is necessary.

Reply:
I just followed the example on line 63 and 68. Do you think we should still change that or leave it for consistency?

Reply:
I think if this is the case, we can just change the previous example as well for simplicity

Reply:
Done